src/test/java/org/folio/rest/impl/CheckinReceivingApiTest.java

```diff
@@ -384,7 +379,7 @@ public void testPostReceivingElectronicPartially() {
     polUpdates.forEach(pol -> {
       PoLine poLine = pol.mapTo(PoLine.class);
       assertThat(poLine.getReceiptStatus(), is(PoLine.ReceiptStatus.PARTIALLY_RECEIVED));
-      assertThat(poLine.getReceiptDate(), is(nullValue()));
```

Comment:
This check should remain as is once comment for `updatePoLineReceiptStatus` method from `HelperUtils.java` is addressed